elements/rh-alert/docs/30-code.md

```diff
@@ -1,10 +1,23 @@
+{% section %}
 {% renderOverview %}{% endrenderOverview %}
```

Comment:
@bennypowers, is there a way to remove the "Overview" copy but keep the "Installation" part?

Reply:
overview comes from the manifest description field - not something I'd like to remove, because it's useful in other contexts like IDE plugins, or wysiwyg listings, etc.
I believe the installation part is generated in the renderOverview shortcode though.

Why do you want to remove the description from the docs page?

Reply:
There should be a description in the Overview tab for every element/pattern, so it feels redundant to add it to the Code tab too. If we have to keep it, where can I go to edit the Code tab's description, so it matches the initial description?

Reply:
you can now use `{% renderInstallation %}{% endrenderInstallation %}`

Reply:
Thank you!